NorthstarDLL/scripts/client/scriptmodmenu.cpp

```diff
@@ -28,7 +28,8 @@ ADD_SQFUNC("bool", NSIsModEnabled, "string modName", "", ScriptContext::SERVER |
 		}
 	}
 
-	return SQRESULT_NULL;
+	g_pSquirrel<context>->pushbool(sqvm, false);
```

Comment:
Just to be sure, this is a fix that's unrelated to displaying incorrectly installed mods, right?

Reply:
Yes, it just fixes the types. The way it currently is, the function returns `bool ornull` instead of `bool`